src/main/java/nl/tudelft/dnainator/parser/buffered/DefaultEdgeParser.java

```diff
@@ -86,22 +107,30 @@ private int eatSpaces(int next) throws IOException {
 	private String parseDest() throws IOException, InvalidEdgeFormatException {
 		StringBuilder dest = new StringBuilder(ID_LENGTH_GUESS);
 		int point = eatSpaces(br.read());
+		boolean destParsed = false;
```

Comment:
Not really fond of this solution, but should not be a reason to block merging imo.